zmsadmin/js/page/availabilityDay/index.js

```
@@ -101,36 +106,55 @@ class AvailabilityPage extends Component {
-                return sendAvailability;
-            }).map(cleanupAvailabilityForSave)
-                
-            console.log('Saving updates', sendData)
+    
+            const selectedDate = formatTimestampDate(this.props.timestamp);
+    
+            const sendData = this.state.availabilitylist
+                .filter((availability) => {
+                    return (
+                        (availability.__modified || 
+                        (availability.tempId && availability.tempId.includes('__temp__'))) &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)